datacats/cli/purge.py

```diff
@@ -45,9 +45,8 @@ def purge(opts):
     environment.purge_data(sites)
 
     if opts['--delete-environment']:
-        if not environment.target:
+        if environment.target:
+            rmtree(environment.target)
+        else:
             print 'Failed to load environment.',
```

Comment:
My only hesitation is that this is more of a polite warning than an "error". but I guess that's ok if it's going to be printed the same way